src/library/templates/library/partial/book_starred.html

```diff
@@ -0,0 +1,43 @@
+{% load static library %}
+
+{% if starred|lower == 'true' %}
```

Comment:
You are passing a boolean so this can just be {% if starred %}

Reply:
Modified to improve if using boolean